### DISCUSSION TITLE
Commit navigation in commit view (arrows next, previous)

### DISCUSSION THREAD

**Discussion by klaus-trabalhando** [community/community#110223] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hi, I think it should be very nice to have a commit navigation, I mean, when we are in the commit details (from commits history), if we want to analyse each commit, we have to get back to commits history and click next/previous, we have also to remember what commit we saw before, instead of this, should be nice in commit view page some arrows to move next or previous.

**Comment by ettaboyle** [comment]
We appreciate your feedback and your valuable input. We want to be transparent about our current priorities. You can gain insights into the projects we’re currently focusing on for the upcoming year by checking our [public roadmap](https://github.com/github/roadmap/projects/1). While smaller fixes may not always be explicitly listed, they often get addressed when related work is in progress.

Regarding your specific request, it’s important to note that it’s not actively being worked on at the moment. However, this doesn’t mean we won’t consider it in the future; it’s simply not on our development roadmap right now.

We’d like to emphasize that as a valued GitHub user, your feedback in this Discussion space, along with appropriate tags and examples, plays a crucial role in shaping the development of GitHub. While we can’t guarantee immediate action on every piece of feedback, please rest assured that we are attentive to your suggestions and continuously strive to enhance the GitHub experience.

While we do review all incoming feedback unfortunately due to the volume of feedback, we may not be able to respond to each one individually at this time. However, we strongly encourage you to continue providing your valuable input as it greatly helps us prioritize and improve GitHub.

Your contributions are instrumental in making GitHub better for all users. Thank you for taking the time to share your thoughts with us, even if the particular request is not currently in active development. Your input is truly valuable and contributes to the continuous improvement of GitHub.
